### ISSUE TITLE
Enable Parallel Execution of Workflow while Maintaining Sequential Execution of Steps in the folder

### ISSUE THREAD

**Issue by ashwinichannaveerappa** [a85/POSTMan-Chrome-Extension#11988]
### Is there an existing request for this feature?

- [X] I have searched the existing issues for this feature request and I know that duplicates will be closed

### Is your feature request related to a problem?

"I would like to request a feature that enables running the complete workflow in parallel while ensuring that the multiple requests in the folder are executed sequentially. 

Currently each requests in the folder will be executed parallelly.

### Describe the solution you'd like

Basically should supports the sequential execution of requests within each thread group. 

This feature would greatly enhance the performance testing capabilities by allowing the execution of the entire workflow to be optimized for parallel execution."

**Comment by malvikach** [comment]
@ashwinichannaveerappa Currently, this is precisely what is happening with the performance testing feature. If I have understood this correctly you want a collection's request to run in a sequential manner by each Virtual user. However multiple Virtual users will run this in parallel hence testing the workflows in parallel and not just the individual requests. 

Let me know what you think.

**Comment by ashwinichannaveerappa** [comment]
@malvikach yeah I would like to run collections requests in a sequential manner by each virtual user. 
Multiple Virtual users will run this in parallel hence testing the workflows in Parallel.

**Comment by malvika-chaudhary** [comment]
Closing this as it is already addressed in the way product works. Please feel free to reopen or create a new feature request if you like to see any other improvements to the feature!

Thanks
Malvika
